cmake/generic.cmake

```diff
@@ -257,31 +258,50 @@ file(MAKE_DIRECTORY ${GOPATH})
 #   tensor # Because ops depend on tensor, this line is optional.
```

Comment:
Can you update the comment, currently it's
```text
# go_library(api
#   SRCS	
#   api.go
#   DEPS
```
I think we don't need to provide api.go anymore. Could you mention the use of STATIC and SHARED as well?